tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Call pytest for testing suite
+pytest -v --disable-pytest-warnings \
+
+# Show coverage report in the terminal
+--cov-report term --cov=covid19 \
+
+# Show which statements are missing coverage
+--cov-report=term-missing \
```

Comment:
iirc you dont need the top `--cov-report term` since you've defined the cov-report again here and it overrides the previous values

Reply:
Ah okay, that makes sense. Removed that line.